### PR TITLE
Update ancient wisdom to be politically correct

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
This updates ancient wisdom (circa 2016) which used outdated pronouns which do not meet the standards of a modern PC linter.

Before
----------------------------------------
Hu knows?

After
----------------------------------------
Hu still knows, but is inclusive and accessible to all beings.

Technical Details
----------------------------------------
Also removed an unused variable. Hu knows why that was there.

### CODE REVIEW

File: CRM/Contact/BAO/Query.php [hunk 1]
```diff
@@ -6355,15 +6355,14 @@ public static function processSpecialFormValue(&$formValues, $specialFields, $ch
    * @param string|CRM_Utils_Sort $sort
    *   The order by string.
    * @param null $sortOrder
-   *   Who knows? Hu knows. He who knows Hu knows who.
+   *   Who knows? Hu knows. One who knows Hu knows who.
```
Comment (totten) [inline CRM/Contact/BAO/Query.php:6358]: I feel like this formulation needs the word "won" as well.

Comment (colemanw) [reply CRM/Contact/BAO/Query.php:6358]: Hu knows Won?

Comment (colemanw) [reply CRM/Contact/BAO/Query.php:6358]: If Won fought Hu... who won?

File: CRM/Contact/BAO/Query.php [hunk 2]
```diff
@@ -6355,15 +6355,14 @@ public static function processSpecialFormValue(&$formValues, $specialFields, $ch
...
    *
    * @return string
    *   list(string $orderByClause, string $additionalFromClause).
    *
    * @throws \CRM_Core_Exception
    */
   protected function prepareOrderBy($sort, $sortOrder) {
-    $orderByArray = [];
```
Comment (composerjk) [inline CRM/Contact/BAO/Query.php:6366]: Was this seemingly unrelated change intended as part of this commit?

Comment (colemanw) [reply CRM/Contact/BAO/Query.php:6366]: Yes, see **Technical Details** of the description :)

Comment (composerjk) [reply CRM/Contact/BAO/Query.php:6366]: Ha, I forgot to read. :) Oops.
